pymc/tests/test_distributions_moments.py

```diff
@@ -751,6 +752,23 @@ def test_categorical_moment(p, size, expected):
     assert_moment_is_expected(model, expected)
 
 
+@pytest.mark.parametrize(
+    "mu, cov, size, expected",
+    [
+        (np.ones(1), np.identity(1), None, np.ones(1)),
+        (np.ones(10), np.identity(10), None, np.ones(10)),
+        (np.ones(2), np.identity(2), 4, np.ones((4, 2))),
+        (np.ones(2), np.identity(2), (4, 2), np.ones((4, 2, 2))),
+        (np.ones((2, 2)), np.identity(2), None, np.ones((2, 2))),
+        (np.ones((2, 2)), np.identity(2), 4, np.ones((4, 2, 2))),
+        (np.ones((2, 2)), np.identity(2), (4, 2), np.ones((4, 2, 2, 2))),
+    ],
+)
+def test_mv_normal_moment(mu, cov, size, expected):
```

Comment:
Looks great. One last suggestion, should we make some of the tests have different means other than 1? For instance `mu=np.array([[1, 2], [3, 4]])`

Reply:
I think that mu should be 1-dim in MvNormal. 
I will add cases where the mu is different along different dimensions.

Reply:
What do you mean about mu? This seems to be perfectly valid: 

```python
x = pm.MvNormal.dist(mu=np.array([[10, 50], [-10, -50]]), cov=np.eye(2)*1)
x.eval()
# array([[  9.85603189,  50.71401641],
#       [ -8.88839562, -49.91483633]])
```

If we don't have that restriction in `pm.MvNormal.dist` or `.logp` we cannot have it in `get_moment` either.